docs/troubleshooting.md

```diff
@@ -172,34 +172,26 @@ token that is required to authenticate with the API server.
 Verify with the following commands:
 
 ```console
-# start a container that contains curl
-$ kubectl run test --image=tutum/curl -- sleep 10000
+# get service IP of master
+$ kubectl get services kuberentes
+NAME         CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+kubernetes   10.100.0.1     <none>        443/TCP   1d
 
-# check that container is running
-$ kubectl get pods
-NAME                   READY     STATUS    RESTARTS   AGE
-test-701078429-s5kca   1/1       Running   0          16s
+# start a container that contains curl
+$ kubectl run -i --tty --rm test --image=tutum/curl --restart=Never
```

Comment:
`tutum/curl` docker image is invalid.
Please using **official docker image for curl** https://hub.docker.com/r/curlimages/curl

Reply:
Using recommended curl image now.